src/libraries/System.Xml.ReaderWriter/ref/System.Xml.ReaderWriter.cs

```diff
@@ -396,31 +407,31 @@ public partial class XmlElement : System.Xml.XmlLinkedNode
+        public override System.Xml.XmlNode? ParentNode { get { throw null; } }
         public override string Prefix { get { throw null; } set { } }
         public override System.Xml.Schema.IXmlSchemaInfo SchemaInfo { get { throw null; } }
         public override System.Xml.XmlNode CloneNode(bool deep) { throw null; }
         public virtual string GetAttribute(string name) { throw null; }
-        public virtual string GetAttribute(string localName, string namespaceURI) { throw null; }
-        public virtual System.Xml.XmlAttribute GetAttributeNode(string name) { throw null; }
-        public virtual System.Xml.XmlAttribute GetAttributeNode(string localName, string namespaceURI) { throw null; }
+        public virtual string GetAttribute(string localName, string? namespaceURI) { throw null; }
+        public virtual System.Xml.XmlAttribute? GetAttributeNode(string name) { throw null; }
+        public virtual System.Xml.XmlAttribute? GetAttributeNode(string localName, string? namespaceURI) { throw null; }
         public virtual System.Xml.XmlNodeList GetElementsByTagName(string name) { throw null; }
         public virtual System.Xml.XmlNodeList GetElementsByTagName(string localName, string namespaceURI) { throw null; }
         public virtual bool HasAttribute(string name) { throw null; }
-        public virtual bool HasAttribute(string localName, string namespaceURI) { throw null; }
+        public virtual bool HasAttribute(string localName, string? namespaceURI) { throw null; }
         public override void RemoveAll() { }
         public virtual void RemoveAllAttributes() { }
         public virtual void RemoveAttribute(string name) { }
-        public virtual void RemoveAttribute(string localName, string namespaceURI) { }
-        public virtual System.Xml.XmlNode RemoveAttributeAt(int i) { throw null; }
-        public virtual System.Xml.XmlAttribute RemoveAttributeNode(string localName, string namespaceURI) { throw null; }
-        public virtual System.Xml.XmlAttribute RemoveAttributeNode(System.Xml.XmlAttribute oldAttr) { throw null; }
+        public virtual void RemoveAttribute(string localName, string? namespaceURI) { }
+        public virtual System.Xml.XmlNode? RemoveAttributeAt(int i) { throw null; }
+        public virtual System.Xml.XmlAttribute? RemoveAttributeNode(string localName, string? namespaceURI) { throw null; }
+        public virtual System.Xml.XmlAttribute? RemoveAttributeNode(System.Xml.XmlAttribute oldAttr) { throw null; }
         public virtual void SetAttribute(string name, string value) { }
-        public virtual string SetAttribute(string localName, string namespaceURI, string value) { throw null; }
-        public virtual System.Xml.XmlAttribute SetAttributeNode(string localName, string namespaceURI) { throw null; }
+        public virtual string SetAttribute(string localName, string? namespaceURI, string value) { throw null; }
```

Comment:
```suggestion
        [return: NotNullIfNotNull("value")]
        public virtual string? SetAttribute(string localName, string? namespaceURI, string? value) { throw null; }
```

I think this API should look like this instead. That also solves the following issue:
```
Possible null reference argument for parameter 'value' in 'string XmlElement.SetAttribute(string localName, string? namespaceURI, string value)'.
```
https://github.com/dotnet/runtime/blob/53f4c08b5442733e81e82d85646f0e4cacf7bae8/src/libraries/System.Data.Common/src/System/Data/xmlsaver.cs#L199

Reply:
I think this is already fixed

Reply:
I think this is already fixed